tests/ExceptionHandlerTest.php

```diff
@@ -124,10 +124,9 @@ public function testReportInvalidReporterClass()
 
         $property->setValue($handler, $config);
 
-        $this->setExpectedException(
-            \InvalidArgumentException::class,
-            'invalid: stdClass is not a valid reporter class.'
-        );
+        $this->expectException(\InvalidArgumentException::class);
+
+        $this->expectExceptionMessage('invalid: stdClass is not a valid reporter class.');
```

Comment:
setExpectedException is no longer supported on phpUnit 8, so replaced by the equivalent code